tests/performance/crypto.spec.ts

```diff
@@ -70,7 +70,8 @@ export default () => {
 
         [false, true].forEach((useSharedKeyStore) => {
             describe(`Web Worker (shared key store=${useSharedKeyStore}, transferables=false)`, () => {
-                it(`encrypt ${Config.CRYPTO_ITERATIONS} times`, (done) => {
+                it(`encrypt ${Config.CRYPTO_ITERATIONS} times`, (done: any) => {
+                    // @ts-ignore
```

Comment:
The linter doesn't check the tests but it is a lint error IntelliJ complains about. Reason being that apparently `window.Worker` is not defined. I guess it can be replaced with just `Worker`.